includes/definitions/msl.yaml

```diff
@@ -0,0 +1,20 @@
+os: msl
+text: 'Mitel Standard Linux'
+type: server
+over:
+    - { graph: device_processor, text: 'Processor Usage' }
+    - { graph: device_ucd_memory, text: 'Memory Usage' }
+    - { graph: device_storage, text: 'Storage Usage' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.1027.1.6.1
+processor_stacked: 1
+poller_modules:
+    bgp-peers: 0
+    ospf: 0
+    stp: 0
+discovery_modules:
+    bgp-peers: 0
+    stp: 0
+    vmware-vminfo: 1
```

Comment:
why do you have these vminfo modules enabled?

Reply:
I based the definitions file off of the Linux one. Guess it would've made sense for me to actually see what that module did.

```
SNMP[/usr/bin/snmpbulkwalk -v2c -c 'public' -OQUs -m +VMWARE-ROOT-MIB:VMWARE-VMINFO-MIB -M /opt/librenms/mibs:/opt/librenms/mibs/vmware udp:msl-host:161 vmwVmTable]
vmwVmTable = No Such Object available on this agent at this OID
```

Guess I'll be disabling that.